### PR TITLE
Introduce xvcfsubsample input prep notebook

### PR DESCRIPTION
**Pull request by xbrianh**

### CODE REVIEW

File: notebooks/xvcfsubsample/main.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    from terra_notebook_utils import table
+
+    workspace = os.environ['WORKSPACE_NAME']
+    workspace_namespace = os.environ['GOOGLE_PROJECT']
+    workspace_bucket = os.environ['WORKSPACE_BUCKET']
+
+with herzog.Cell("markdown"):
+    """
+    ## Option A: VCF subsample workflow input for DRS URIs
+    This is a typical workflow preparation for merging TOPMed VCFs _without_ downloading them to your workspace.
```
Comment (bethsheets) [inline notebooks/xvcfsubsample/main.py:45]: replace merging with subsampling on this line.

File: notebooks/xvcfsubsample/main.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    """
+
+table.delete("vcf-subsample-input-drs")
+with herzog.Cell("python"):
+    table_name = "vcf-subsample-input-drs"
+    table.put_row(table_name, dict(workspace=workspace,
+                                   billing_project=workspace_namespace,
+                                   input="drs://dg.4503/b2871873-8dcb-4a3e-a926-a17ab4a19f0a",
+                                   output=f"{workspace_bucket}/subsampled/drs_subsampled_a.vcf.gz",
+                                   samples=["NWD999037", "NWD996859"]))
```
Comment (bethsheets) [inline notebooks/xvcfsubsample/main.py:56]: Could we provide a commented-out alternative to import samples from a file?

**Review by bethsheets — CHANGES_REQUESTED** [review]
Looks good! Just a couple of comments.

File: notebooks/xvcfsubsample/main.py (hunk 1 above)
Comment (xbrianh) [reply notebooks/xvcfsubsample/main.py:45]: done

File: notebooks/xvcfsubsample/main.py (hunk 2 above)
Comment (xbrianh) [reply notebooks/xvcfsubsample/main.py:56]: Samples may conceivably be stored in several formats such as CSVs, text files, and json. This makes it difficult to create a good example.

However, I've included some language describing how to localize a file to the notebook and open it. Parsing out the content is an exercise left to the reader. Let me know what you think.

Comment (bethsheets) [reply notebooks/xvcfsubsample/main.py:56]: That's totally fair. I like your approach here!

**Review by bethsheets — APPROVED** [review]

**Pull request merged**